site/en/docs/extensions/mv3/i18n-messages/index.md

```diff
@@ -16,6 +16,7 @@ how to internationalize and localize, see the [Internationalization][1] page.
 The following code shows the supported fields for `messages.json`. Only the "_name_" and "message"
 fields are required.
 
+{% Label %}messages.json:{% endLabel %}
```

Comment:
```suggestion
{% Label %}messages.json:{% endLabel %}

```
These need an extra line between for them to render correctly. Can you add the extra line throughout?

Reply:
It seems to me that the space and the colon are making up for a lack of distinct formatting. I've opened a ticket on this (#5785). I'm going to see if I can get Rachel to expedite it, since it is likely a small job. Can we hold off on doing anything with these and be prepared to remove the colons. I hope to have an answer from Rachel by tomorrow or Monday.

Reply:
I don't see a problem with adding an extra line between 🤷🏻‍♀️

Reply:
Yep, you need to add the extra line. I don't think that's an issue to do (other than something you have to remember to do). There are a couple of other instances where this happens iirc.